src/views/HomeView/index.js

```diff
@@ -3,7 +3,7 @@ import React, { PropTypes } from 'react'
 import { connect } from 'react-redux'
 import { increment, doubleAsync } from '../../redux/modules/counter'
 import DuckImage from './Duck.jpg'
-import classes from './HomeView.scss'
+import classes from './style.scss'
```

Comment:
The reason I avoid this is because it produces less-than-desirable class names w/ CSS modules.

Compare: `HomeView__<class>__<hash>` (changing `HomeView` to whatever component is being used`) vs. `style__<class>__<hash>`.